gui/src/components/Backups.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+    <div>
+        <h4 class="pt-3">Backups in Cluster: {{cluster.cluster_name}}</h4>
+        <ClusterSummary :cluster="cluster"/>
+        <div class="row justify-content-end mb-3 pr-3">
+            <button class="btn btn-primary mx-1" data-toggle="modal" data-target="#registerBackup">Create Backup
```

Comment:
I'm not sure, but should it be like this ?
```suggestion
            <button class="btn btn-primary mx-1" data-toggle="modal" data-target="#registerBackup">Create a Backup
```
or `Create Backups` ?

Reply:
It can be as you suggested if you prefer. I think usually buttons like this can omit the "a".

Reply:
OK ! Then no `a`. (keeping it as it is is fine)